src/main/java/io/github/seggan/slimypowersuits/SuitUtils.java

```diff
@@ -33,16 +33,18 @@ private SuitUtils() {}
         List<String> lore = meta.getLore();
         if (lore == null) {
             return new ArrayList<>();
+        } else {
+            return getInstalledModules(lore);
         }
-        return getInstalledModules(lore);
     }
 
     public static boolean isPowerSuitPiece(ItemStack armor) {
         SlimefunItem item = SlimefunItem.getByItem(armor);
-        if (item == null) {
+        if (item != null) {
+            return item instanceof SuitPiece;
+        } else {
             return false;
         }
```

Comment:
This is completely redundant.

An instanceof check is already an extension of a null check since null isn't an instance of anything.

```suggestion
        return item instanceof SuitPiece;
```